articles/custom-domains/feature-specific-setup.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Feature Specific Setup with Custom Domains
+description: How to set up custom domains with feature specific instructions
+toc: true
+---
+# Feature-Specific Setup
```

Comment:
The title is way too generic, I had to look into the ToC to understand what this doc is about. What if you changed it to sth like "Additional Configuration for Custom Domains" or sth like that? Then at the first sentence you can explain in more detail what this doc is about.